actionview/test/actionpack/abstract/layouts_test.rb

```diff
@@ -295,7 +295,7 @@ class TestBase < ActiveSupport::TestCase
         10.times do |x|
           controller = WithString.new
           controller.define_singleton_method :index do
-            render template: ActionView::Template::Text.new("Hello string!"), locals: { :"x#{x}" => :omg }
+            render template: ActionView::Template::Text.new("Hello string!"), locals: { "x#{x}" => :omg }
```

Comment:
Which cop does this violate?

Reply:
`Layout/EmptyLinesAroundBlockBody` -> https://github.com/rails/rails/blob/master/.rubocop.yml#L65

This was changed in https://github.com/rubocop-hq/rubocop/pull/7090 I think

Reply:
I asked why rubocop correct intentional symbol key to string key wrongly.

Reply:
@kamipo : My bad. I wanted to comment it for another comment of your(https://github.com/rails/rails/pull/36426#discussion_r291054490).

`Style/HashSyntax` cop fails here. It should be following as per -> https://github.com/rubocop-hq/rubocop/pull/6948

Rubocop new syntax when keys are interpolated.

```
"x#{x}": :omg
```

I'll make that change.

Reply:
This difference is due to the following changes.
https://github.com/rubocop-hq/rubocop/pull/6948